static/specification.html

```diff
@@ -59,384 +58,130 @@
-          <a href="./docs/explainer">explainer</a> for more info.
+          best shape it should take. Please see the <a href=
+          "./docs/explainer">explainer</a> for more info.
         </p>
         <p>
-          The specification reflects the desired end-state of the Web Monetization
-          APIs as currently anticipated and agreed to between the contributors.
-          The specification is being prepared here, in this format, to collect the
-          input of the Web community and prepare the work to ultimately follow the
-          W3C standards track should it have the necessary support to do so.
+          The specification reflects the desired end-state of the Web
+          Monetization APIs as currently anticipated and agreed to between the
+          contributors. The specification is being prepared here, in this
+          format, to collect the input of the Web community and prepare the
+          work to ultimately follow the W3C standards track should it have the
+          necessary support to do so.
         </p>
         <p>
-          For the most accurate reflection of the APIs that have been implemented
-          by providers see the <a href="./docs/api">API documentation</a>.
+          For the most accurate reflection of the APIs that have been
+          implemented by providers see the <a href="./docs/api">API
+          documentation</a>.
         </p>
       </div>
     </section>
     <section id="terminology">
-      <h2>Terminology</h2>
+      <h2>
+        Terminology
+      </h2>
       <dl>
-        <dt><dfn>User</dfn>:</dt>
-        <dd>the party who is accessing the web-monetized website.</dd>
-        <dt><dfn>Website</dfn>:</dt>
-        <dd>the web-monetized website.</dd>
-        <dt><dfn data-lt="provider">Web Monetization provider</dfn>:</dt>
-        <dd>the party making payments on behalf of the user.</dd>
-        <dt><dfn>Web Monetization receiver</dfn>:</dt>
-        <dd>the party receiving payments on behalf of the website.</dd>
-        <dt><dfn>User agent</dfn>:</dt>
-        <dd>the web browser used by the user.</dd>
+        <dt>
+          <dfn data-lt="provider">Web Monetization provider</dfn>:
+        </dt>
+        <dd>
+          The party making payments on behalf of the user. A Web Monetization
+          provider can, for example, make use of the Simple Payment Setup
+          Protocol (<abbr title="Simple Payment Setup Protocol">SPSP</abbr>) on
+          top of the Interledger Protocol (<abbr title=
+          "">ILP</abbr>)/[[[STREAM]]] to provide a high-level way to send money
+          and data.
+        </dd>
+        <dt>
+          <dfn>Web Monetization receiver</dfn>:
+        </dt>
+        <dd>
+          The party receiving payments on behalf of the website.
+        </dd>
+        <dt>
+          <dfn>Payment end-point</dfn>:
+        </dt>
+        <dd>
+          A [=URL=] to an [[[OpenPayments]]] conforming HTTP end-point.
```

Comment:
That's an ["SPSP Endpoint"](https://interledger.org/rfcs/0009-simple-payment-setup-protocol/#specification) according to the ILP spec. Maybe we should change the Open Payments mention to SPSP.